### ISSUE TITLE
Parameters are misplaced for polymorphic types

### ISSUE THREAD

**Issue by t03apt** [domaindrivendev/Swashbuckle.AspNetCore#1544]
# Description

Parameters are misplaced for polymorphic types.

### C# model
https://github.com/domaindrivendev/Swashbuckle.AspNetCore/blob/6fd662b7e1f1a9920dcd5a0f5d71bbe8f0c76187/test/WebSites/Basic/Controllers/PolymorphicTypesController.cs#L23-L38

### Currently `Circle` model is generated like this
```[json]
      "Circle": {
        "type": "object",
        "allOf": [
          {
            "$ref": "#/components/schemas/Shape"
          }
        ],
        "properties": {
          "radius": {
            "type": "integer",
            "format": "int32"
          }
        },
        "additionalProperties": false
      },
```

### Expected value according to [OpenAPI docs (see: Cat class)](https://swagger.io/docs/specification/data-models/oneof-anyof-allof-not/#allof):
```[json]
      "Circle": {
        "allOf": [
          {
            "$ref": "#/components/schemas/Shape"
          },
          {
            "type": "object",
            "properties": {
              "radius": {
                "type": "integer",
                "format": "int32"
              }
            },
            "additionalProperties": false
          }
        ]
      },
```

**Comment by t03apt** [comment]
Related: https://github.com/cyclosproject/ng-openapi-gen/issues/74

**Comment by domaindrivendev** [comment]
Addressed by 67c27146d8b1e98f1704e547a0d21a20586d2cc5
